### ISSUE TITLE
PyTorch 2.10.0 Added a Check for Epsilon > 0.0, Which Breaks BatchNorm Merging

### ISSUE THREAD

**Issue by lecode-official** [chr5tphr/zennit#234]
The `MergeBatchNorm` canonizer sets `eps` to 0 to turn the `_BatchNorm` instance into an identity function. Unfortunately, the latest PyTorch version (2.10.0) adds a check for `eps > 0.0` and raises an exception when `eps` is less than or equal to 0.0.

**Comment by chr5tphr** [comment]
Hey David,

Thank you for the issue! That's a bummer, I will think of something. Do you already have a workaround that you used?

**Comment by lecode-official** [comment]
No, I reverted back to the previous version of PyTorch. And I don’t think there is anything that can be done about it, except for maybe monkey-patching the relevant setter method?
